gen.go

```diff
@@ -47,6 +47,16 @@ func genPkg(odir string, p *bind.Package, lang string) error {
 		pyvers = 2
 	case "python3", "py3":
 		pyvers = 3
+	case "go", "cffi":
+		pylang, err := getPythonVersion()
```

Comment:
if `getPythonVersion()` fails for some reason, we shadow the `err` variable from earlier.

you should do:
```go
var pylang string
pylang, err = getPythonVersion()
```

Reply:
thanks. I fix it.

Reply:
done